docs/ldml/tr35-keyboards.md

```diff
@@ -2347,6 +2643,22 @@ Another partial example allows a keyboard implementation to prevent people typin
 </transformGroup>
 ```
 
+#### Reorder and Markers
+
+Markers are not matched by `reorder` elements. However, if a character preceded by one or more markers is reordered due to a `reorder` element, those markers will be reordered with the characters, maintaining the same relative order.  This is a similar process to the algorithm used to normalize strings processed by `transform` elements.
+
+Keyboard implementations must process reorders using the following algorithm.
```

Comment:
You fixed 'reorders' in one place but not others.

Reply:
done